src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/smart-list/smart-list.component.ts

```diff
@@ -194,7 +194,7 @@ smartmontools is required to successfully retrieve data.`;
         prop: 'errors_corrected_by_rereads_rewrites',
         name: $localize`Errors Corrected by Rereads/Rewrites`
       },
-      { prop: 'gigabytes_processed', name: $localize`Gigabyes Processed` },
+      { prop: 'gigabytes_processed', name: $localize`Gigabytes Processed` },
```

Comment:
this is likely to break localizations

Reply:
Might be, but it'd be a wrong i18n, so better to have it fixed :D